test/components/preprocessors/test_recursive_splitter.py

```diff
@@ -426,7 +426,7 @@ def test_run_separator_exists_but_split_length_too_small_fall_back_to_character_
     result = splitter.run(documents=[doc])
     assert len(result["documents"]) == 10
     for doc in result["documents"]:
-        if re.escape(doc.content) not in ["\ "]:
+        if re.escape(doc.content) not in ["\\ "]:
```

Comment:
I remember seeing a warning about a regex. I assume this helps to get rid of that warning. 👍

Reply:
yes, that's it 👍🏽